tutorials/applications/dask.md

```diff
@@ -101,10 +101,9 @@ and handing a `numpy` array to `napari` each time a new timepoint or channel is
 ```python
 import napari
 
-with napari.gui_qt():
-    # specify contrast_limits and is_pyramid=False with big data
-    # to avoid unnecessary computations
-    napari.view_image(stack, contrast_limits=[0,2000], multiscale=False)
+# specify contrast_limits and is_pyramid=False with big data
+# to avoid unnecessary computations
+napari.view_image(stack, contrast_limits=[0,2000], multiscale=False)
 ```
 
 *Note: providing the* `contrast_limits` *and* `multiscale` *arguments prevents* `napari` *from trying to calculate the data min/max, which can take an extremely long time with big data.
```

Comment:
```suggestion
*Note: providing the* `contrast_limits` *and* `multiscale` *arguments prevents* `napari` *from trying to calculate the data min/max, which can take an extremely long time with big data.*
```

Reply:
I tried it, but that cuts out the italics from the next sentence "See napari issue #736 for further discussion.", which i don't think we want here